.travis.yml

```diff
@@ -31,8 +31,9 @@ matrix:
       env:
         - DEPENDS="cython numpy scipy matplotlib h5py nibabel cvxopt scikit_learn"
 before_install:
+    - apt-get update && apt-get install -y python-vtk
```

Comment:
Yes - I am just reading through the docs about this.

Sadly, it seems that we will also need to ask for travis to white-list vtk:
https://github.com/travis-ci/apt-source-whitelist



On Tue, Oct 13, 2015 at 5:59 PM, Matthew Brett <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/nipy/dipy/pull/732#discussion_r41945601>:
>
> > @@ -31,8 +31,9 @@ matrix:
> >        env:
> >          - DEPENDS="cython numpy scipy matplotlib h5py nibabel cvxopt scikit_learn"
> >  before_install:
> > +    - apt-get update && apt-get install -y python-vtk
>
> I believe you'll need sudo for the apt-get. I think you can use sudo for
> selected builds - but at the moment the global setting is sudo: false -
> which makes the builds faster. If you want python-vtk for all builds, you
> can add it to the apt package list - e.g
>
> addons:
>   apt:
>     packages:
>       - python-vtk
>
> See the matplotlib .travis.yml file for an example.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/732/files#r41945601>.
>